core/src/main/resources/application.properties

```diff
@@ -33,6 +33,10 @@ spring.jackson.deserialization.fail-on-reading-dup-tree-key=true
 spring.jackson.deserialization.fail-on-ignored-properties=true
 spring.jackson.deserialization.accept-float-as-int=false
 spring.jackson.serialization.write-dates-as-timestamps=false
+spring.jackson.serialization.indent_output=true
+
+# Management (Actuator)
+management.security.enabled=false
```

Comment:
Maybe add a comment this can be done because the ports are not accessible from outside.